### DISCUSSION TITLE
Display last committed time for files

### DISCUSSION THREAD

**Discussion by aivarannamaa** [eza-community/eza#1036] [Q&A, Howto, Help] [answered]
I found eza while looking for a tool, which can list files together with their last commit time (I find `git ls-files | while read file; do echo "$(git log -1 --format="%cd" --date=iso -- "$file") $file"; done` a bit verbose).

Can `eza` do this?

**Comment by gierens** [comment, marked as answer]
Hi @aivarannamaa 

No, the different git options are all focused on displaying the git status only either for files and repository directories.

But feel free to open a feature request issue for this.
